vassal-app/src/main/java/VASSAL/build/module/WizardSupport.java

```diff
@@ -188,7 +188,18 @@ protected Void doInBackground() {
       }
     }
     else {
-      g.getPlayerWindow().setVisible(true);
+      final Boolean showWizard = (Boolean) Prefs.getGlobalPrefs().getValue(WizardSupport.WELCOME_WIZARD_KEY);
+      if (Boolean.TRUE.equals(showWizard)) {
+        if (g.shutDown()) {
+          System.exit(0);
+        }
+        else {
+          g.getPlayerWindow().setVisible(true);
+        }
+      }
+      else {
+        g.getPlayerWindow().setVisible(true);
+      }
```

Comment:
Couldn't you eliminate both `else` cases and stick `g.getPlayerWindow().setVisible(true);` below the end of the outer `if`?

Reply:
In fact, just have one `if`: `if (Boolean.TRUE.equals(showWizard) && g.shutDown())`

Reply:
done

Reply:
Alzo Alzo done